### PR TITLE
[benchmark] Adding typings for event.target

### PR DESCRIPTION
**Pull request by fishcharlie**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://benchmarkjs.com/docs#Event_prototype_target
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by fishcharlie** [comment]
@RyanCavanaugh

@typescript-bot - `Please bother @RyanCavanaugh if I appear to be misbehaving.`

It looks like @typescript-bot didn't update this comment https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44282#issuecomment-620289878 or the labels on this PR now that all checks are now passing.

Anyway to have @typescript-bot relook at this PR (since it didn't automatically)?

**Comment by fishcharlie** [comment]
image

That also doesn't make sense at all...

**Comment by fishcharlie** [comment]
@orta Is the `The Travis CI build failed` supposed to be showing in this case? Even after editing the original comment?

**Comment by orta** [comment]
https://github.com/DefinitelyTyped/dt-mergebot/issues/14

File: types/benchmark/index.d.ts [hunk 1]
```diff
@@ -135,6 +135,40 @@ declare namespace Benchmark {
...
+        fn?: Function;
+        id: number;
+        stats?: {
+            moe?: number;
+            rme?: number;
+            sem?: number;
+            deviation?: number;
+            mean?: number;
+            sample?: any[];
+            variances?: number;
```
Comment (blair) [inline types/benchmark/index.d.ts:159]: If this block isn't replaced with Stats, then `variances` should be `variance`, the word `variances` doesn't appear in benchmark's source code.

File: types/benchmark/index.d.ts [hunk 2]
```diff
@@ -135,6 +135,40 @@ declare namespace Benchmark {
...
+        defer?: boolean;
+        delay?: number;
+        initCount?: number;
+        maxTime?: number;
+        minSamples?: number;
+        minTime?: number;
+        name?: string;
+        fn?: Function;
+        id: number;
+        stats?: {
```
Comment (blair) [inline types/benchmark/index.d.ts:152]: Can you use the Stats type here instead?

**Pull request merged**